game/scripts/npc/abilities/morphling_adaptive_strike_agi.txt

```diff
@@ -41,7 +41,7 @@
       "01"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "damage_base"                                     "55 60 65 70 140 280" //OAA
+        "damage_base"                                     "40 60 80 120 240 480" //OAA
```

Comment:
Adaptive strike agi early on is already hot garbage and morph is not at all a good early game hero that would deserve this nerf.

Reply:
It's a lvl 1 nerf. Dont be dramatic.